cms/static/js/certificates/spec/views/certificate_editor_spec.js

```diff
@@ -90,6 +90,16 @@ function(_, Course, CertificateModel, SignatoryModel, CertificatesCollection, Ce
         uploadDialogTpl = readFixtures('upload-dialog.underscore');
 
         beforeEach(function() {
+            window.course = new Course({
```

Comment:
Can you clarify, here? I think there's a weird merge conflict resolution here which caused the definition of `window.course` to be duplicated twice in the `beforeEach` but besides that it isn't redefined in tests below this describe block.